src/components/AnonymousChat/child/ChatItemText.style.ts

```diff
@@ -0,0 +1,179 @@
+import {StyleSheet} from 'react-native';
+
+import dimen from '../../../utils/dimen';
+import {ANONYMOUS, SIGNED} from '../../../hooks/core/constant';
+import {colors} from '../../../utils/colors';
+import {fonts, normalizeFontSize} from '../../../utils/fonts';
+
+const AVATAR_SIZE = 24;
+const CONTAINER_LEFT_PADDING = 60;
+const CONTAINER_RIGHT_PADDING = 10;
+const AVATAR_LEFT_MARGIN = 8;
+const BUBBLE_LEFT_PADDING = 8;
+const BUBBLE_RIGHT_PADDING = 8;
+
+export const targetLastLine =
+  CONTAINER_LEFT_PADDING -
+  CONTAINER_RIGHT_PADDING -
+  AVATAR_SIZE -
+  AVATAR_LEFT_MARGIN -
+  BUBBLE_LEFT_PADDING -
+  BUBBLE_RIGHT_PADDING;
+
+export const styles = StyleSheet.create({
+  chatContainer: {
+    display: 'flex',
+    marginVertical: dimen.normalizeDimen(3)
+  },
+  chatContainerMyText: {
+    paddingLeft: CONTAINER_LEFT_PADDING,
+    paddingRight: CONTAINER_RIGHT_PADDING
+  },
+  chatContainerTargetText: {
+    paddingLeft: CONTAINER_RIGHT_PADDING,
+    paddingRight: CONTAINER_LEFT_PADDING
+  },
+  chatContainerPromptMyText: {
+    paddingLeft: dimen.normalizeDimen(26)
+  },
+  chatContainerPromptTargetText: {
+    paddingRight: dimen.normalizeDimen(26)
+  },
+  wrapper: {
+    flexDirection: 'row'
+  },
+  chatTitleContainer: {
+    display: 'flex',
+    flexDirection: 'row'
+  },
+  textContainer: {
+    flex: 1,
+    paddingHorizontal: dimen.normalizeDimen(6),
+    paddingVertical: dimen.normalizeDimen(4),
+    backgroundColor: colors.lightgrey,
+    borderRadius: 8
+  },
+  textContainerSigned: {
+    backgroundColor: colors.darkBlue
+  },
+  textContainerAnon: {
+    backgroundColor: colors.anon_primary
+  },
+  textContainerMyText: {
+    borderTopEndRadius: 0
+  },
+  textContainerTargetText: {
+    borderTopStartRadius: 0
+  },
+  textContainerNewLine: {
+    paddingBottom: 14
```

Comment:
Why not all padding and margin not using `dimen.normalizeDimen()` ?

Reply:
the absolute number will be deleted in another ticket because the style will not be used (due to bug fix)